src/bilder/images/edxapp_v2/templates/edxapp/mitxonline/common_values.yml.tmpl

```diff
@@ -384,7 +384,9 @@ INTEGRATED_CHANNELS_API_CHUNK_TRANSMISSION_LIMIT:
 JWT_EXPIRATION: 30
 LANGUAGE_CODE: en
 LANGUAGE_COOKIE: {{ env "ENVIRONMENT" }}-openedx-language-preference
-LEARN_AI_API_URL: https://{{ key "edxapp/learn-ai-api-domain" }}  # Added for ol_openedx_chat
+MIT_LEARN_AI_API_URL: https://{{ key "edxapp/learn-ai-api-domain" }}  # Added for ol_openedx_chat
+MIT_LEARN_API_BASE_URL: https://{{ key "edxapp/mit-learn-api-domain" }}  # Added for ol_openedx_chat
+MIT_LEARN_SUMMARY_FLASHCARD_URL: https://{{ key "edxapp/mit-learn-api-domain" }}/api/v1/contentfiles/  # Added for ol_openedx_chat
```

Comment:
We need to update the keys for both these to point to MIT Learn API URLs. For RC, the Base URL is `https://api.rc.learn.mit.edu`